code/drasil-docLang/lib/Drasil/Sections/SpecificSystemDescription.hs

```diff
@@ -90,10 +90,13 @@ physSystDesc progName parts fg other = SRS.physSyst (intro : bullets : LlC fg :
         bullets = enumSimpleU 1 (short physSyst) parts
 
 -- | General constructor for the Goal Statement section. Takes the given inputs ('Sentence's) and the descriptions ('Contents').
-goalStmtF :: [Sentence] -> [Contents] -> Section
-goalStmtF givenInputs otherContents = SRS.goalStmt (intro:otherContents) []
-  where intro = mkParagraph $ S "Given" +:+ foldlList Comma List givenInputs `sC` 
-                pluralNP (the goalStmt) +: S "are"
+goalStmtF :: [Sentence] -> [Contents] -> Int -> Section
+goalStmtF _ [] _ = SRS.goalStmt ([mkParagraph $ S "Not applicable"]) []
```

Comment:
To fix the HLint error, these parenthesis should be removed 
```suggestion
goalStmtF _ [] _ = SRS.goalStmt [mkParagraph $ S "Not applicable"] []
```
Other than that, this looks good to me!